code/modules/unit_tests/outfit_datums.dm

```diff
@@ -0,0 +1,11 @@
+/datum/unit_test/outfit_datums
+
+// Make sure that the slots indexes used by outfit datums are indeed slot indexes.
+
+/datum/unit_test/outfit_datums/start()
+    for (var/type in subtypesof(/datum/outfit))
+        var/datum/outfit/O = new
+        for (var/L in O.items_to_spawn)
+            for (var/slot in L)
+                if (text2num(slot) == "")
```

Comment:
I think this test is broken because text2num should return null on failure, not empty string.

Reply:
Correct, worth noting that `text2num` just returns the number if passed one.

Reply:
Also, this isn't the test I wrote, I think it got lost in the conflict solving because I'm dumb.